### PR TITLE
enh(doc): add specific endpoint behaviours to cloud documentation

### PR DESCRIPTION
**Pull request by adr-mo**
This PR intends to update the documentation regarding specific API behaviours in Cloud environments.
Therefore API documentation of the following endpoints were updated (**_and some errors fixed_**)

**Hosts** 
- POST Add a host
- PATCH Partial update a host
- GET All Hosts

**Host groups**
- POST Add a host group
- PATCH Partial update a host group
- GET All Host groups

**Host templates**
- POST Add a host template
- PATCH Partial update a host template
- GET All Host templates

**Services**
- POST Add a service
- PATCH Partial update a service
- GET All Services

**Service templates**
- POST Add a service template
- PATCH Partial update a service template
- GET All Service templates

**Resource Access Management**
Documentation has been moved under `Administration` folder.

**Notification**
All the documentation linked to the "Cloud notification" has been moved in the Cloud API documentation

**Important**
All the endpoints that are available for onPrem and has a specific behaviour in Cloud environement has a special mention in their description

**Fixes** # (issue)

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

#### Community contributors & Centreon team

- [ ] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

## Summary by CodeRabbit

- **Documentation**
	- Updated Centreon Web API documentation by removing outdated endpoint description steps.

### CODE REVIEW

**Review by callapa — DISMISSED** [review]

**Review by wtermellil — APPROVED** [review]

**Review by kduret — APPROVED** [review]

**Review by wtermellil — APPROVED** [review]

File: centreon/doc/API/latest/Cloud/Configuration/HostGroups/FindAndUpdateHostGroup.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  description: |
+    Update an existing host group.
+  parameters:
+    - $ref: 'QueryParameter/HostGroupId.yaml'
+  requestBody:
+    required: true
+    content:
+      application/json:
+        schema:
+          $ref: 'Schema/PartialUpdateHostGroup.yaml'
```
Comment (a-launois) [inline centreon/doc/API/latest/Cloud/Configuration/HostGroups/FindAndUpdateHostGroup.yaml:35]: A PUT is not a partial update

File: centreon/doc/API/latest/Cloud/Configuration/HostGroups/Schema/PartialUpdateHostGroup.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+type: object
```
Comment (a-launois) [inline centreon/doc/API/latest/Cloud/Configuration/HostGroups/Schema/PartialUpdateHostGroup.yaml:1]: File needs renaming (see FindAndUpdateHostGroup.yaml)

**Review by jeremyjaouen — APPROVED** [review]

File: centreon/doc/API/v24.04/Configuration/Host/Schema/FindHostsResponse.yaml [hunk 3]
```diff
@@ -46,7 +46,7 @@ properties:
           type: string
           description: "Host template name"
           example: 'generic-host'
-  normal_check_interval:
+  normal_normal_check_interval:
```
Comment (a-launois) [inline centreon/doc/API/v24.04/Configuration/Host/Schema/FindHostsResponse.yaml:49]:
```suggestion
  normal_check_interval:
```

File: centreon/doc/API/v24.04/Configuration/Service/Schema/AddServiceRequest.yaml [hunk 4]
```diff
@@ -45,7 +45,7 @@ properties:
     type: integer
     description: "Define the number of times that the monitoring engine will retry the service check command if it returns any non-OK state."
     nullable: true
-  normal_check_interval:
+  normal_normal_check_interval:
```
Comment (a-launois) [inline centreon/doc/API/v24.04/Configuration/Service/Schema/AddServiceRequest.yaml:48]:
```suggestion
  normal_check_interval:
```

File: centreon/doc/API/v24.04/Configuration/Service/Schema/AddServiceResponse.yaml [hunk 5]
```diff
@@ -49,7 +49,7 @@ allOf:
         type: integer
         description: "Define the number of times that the monitoring engine will retry the service check command if it returns any non-OK state."
         nullable: true
-      normal_check_interval:
+      normal_normal_check_interval:
```
Comment (a-launois) [inline centreon/doc/API/v24.04/Configuration/Service/Schema/AddServiceResponse.yaml:52]:
```suggestion
      normal_check_interval:
```

File: centreon/doc/API/v24.04/Configuration/Service/Schema/FindServicesResponse.yaml [hunk 6]
```diff
@@ -57,7 +57,7 @@ allOf:
             host_name:
               type: string
               example: 'host-name'
-      normal_check_interval:
+      normal_normal_check_interval:
```
Comment (a-launois) [inline centreon/doc/API/v24.04/Configuration/Service/Schema/FindServicesResponse.yaml:60]:
```suggestion
      normal_check_interval:
```

File: centreon/doc/API/v24.04/Configuration/Service/Schema/PartialUpdateServiceRequest.yaml [hunk 7]
```diff
@@ -45,7 +45,7 @@ properties:
     type: integer
     description: "Define the number of times that the monitoring engine will retry the service check command if it returns any non-OK state."
     nullable: true
-  normal_check_interval:
+  normal_normal_check_interval:
```
Comment (a-launois) [inline centreon/doc/API/v24.04/Configuration/Service/Schema/PartialUpdateServiceRequest.yaml:48]:
```suggestion
  normal_check_interval:
```

**Review by tuntoja — APPROVED** [review]

File: centreon/doc/API/v24.04/Configuration/ServiceTemplate/Schema/AddServiceTemplateResponse.yaml [hunk 8]
```diff
@@ -49,7 +49,7 @@ allOf:
         type: integer
         description: "Define the number of times that the monitoring engine will retry the service check command if it returns any non-OK state."
         nullable: true
-      normal_check_interval:
+      normal_normal_check_interval:
```
Comment (a-launois) [inline centreon/doc/API/v24.04/Configuration/ServiceTemplate/Schema/AddServiceTemplateResponse.yaml:52]:
```suggestion
      normal_check_interval:
```

File: centreon/doc/API/v24.04/Configuration/ServiceTemplate/Schema/NewServiceTemplate.yaml [hunk 9]
```diff
@@ -40,7 +40,7 @@ properties:
     type: integer
     description: "Define the number of times that the monitoring engine will retry the service check command if it returns any non-OK state."
     nullable: true
-  normal_check_interval:
+  normal_normal_check_interval:
```
Comment (a-launois) [inline centreon/doc/API/v24.04/Configuration/ServiceTemplate/Schema/NewServiceTemplate.yaml:43]:
```suggestion
  normal_check_interval:
```

File: centreon/doc/API/v24.04/Configuration/ServiceTemplate/Schema/PartialUpdateServiceTemplate.yaml [hunk 10]
```diff
@@ -38,7 +38,7 @@ properties:
   max_check_attempts:
     type: integer
     description: "Define the number of times that the monitoring engine will retry the service check command if it returns any non-OK state."
-  normal_check_interval:
+  normal_normal_check_interval:
```
Comment (a-launois) [inline centreon/doc/API/v24.04/Configuration/ServiceTemplate/Schema/PartialUpdateServiceTemplate.yaml:41]:
```suggestion
  normal_check_interval:
```

**Review by a-launois — CHANGES_REQUESTED** [review]

**Pull request merged**